metacontroller/rockcraft.yaml

```diff
@@ -20,17 +20,18 @@ parts:
     plugin: go
     source: https://github.com/metacontroller/metacontroller
     source-type: git
-    source-tag: "v3.0.0"
+    source-tag: "v4.11.22"
     build-snaps:
-      - go/1.18/stable
+      - go/1.23/stable
+      - goreleaser/latest/stable
```

Comment:
Let's add a comment to indicate that this comes from here https://github.com/metacontroller/metacontroller/blob/v4.11.22/Makefile#L22